packages/search/tests/unit/services/navi-search/navi-definition-search-provider-test.js

```diff
@@ -0,0 +1,119 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Pretender from 'pretender';
+import metadataRoutes from '../../../helpers/metadata-routes';
+
+let Service, Server;
+
+module('Unit | Service | navi-definition-search-provider', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    Server = new Pretender(metadataRoutes);
+    Service = this.owner.lookup('service:navi-search/navi-definition-search-provider');
+  });
+
+  hooks.afterEach(async function() {
+    Server.shutdown();
+  });
+
+  test('search definition of a dimension', async function(assert) {
+    assert.expect(3);
+    metadataRoutes.bind(Server);
+    const results = await Service.search.perform('age');
+
+    const expectedResults = [
+      {
+        id: 'age',
+        name: 'Age'
+      }
+    ];
+
+    assert.equal(results.component, 'navi-search-result/definition', 'Result contains correct display component name');
+    assert.equal(results.title, 'Definition', 'Result contains correct title for the search result section');
+    results.data.forEach((result, index) => {
```

Comment:
Is this loop necessary when `expectedResults` only has 1 index?

Reply:
`results.data` is going to be an array anyways I feel a loop is more elegant than having subscript notation `results.data[0]` to compare the one result, and also the code is consistent with the test bellow that returns two results.

Reply:
`assert.deepEqual(results.data ...` would be more elegant than either option

Reply:
but then `results.data` has a bunch more properties that come from the keg and it won't be equal to my expected result.
<img width="582" alt="Screen Shot 2020-04-09 at 3 52 11 PM" src="https://user-images.githubusercontent.com/1649445/78939594-14b18b00-7a7a-11ea-8b9a-c6ffdb9d5634.png">

Reply:
See my comment below